validate.go

```diff
@@ -170,6 +170,17 @@ func checkProcess(process rspec.Process, rootfs string) {
 		}
 	}
 
+	if len(process.Args) < 1 {
+		logrus.Fatalf("args should not be empty")
+	}
+	command := process.Args[0]
+	if filepath.IsAbs(command) {
+		cmdPath := path.Join(rootfs, command)
+		if _, err := os.Stat(cmdPath); err != nil {
+			logrus.Fatalf("Cannot find the command path %q", cmdPath)
```

Comment:
See also my notes [here][1] on the current spec phrasing being inconsistent.  The in-flight opencontainers/runtime-spec#427 should help make the specification more self-consistent.

[1]: https://github.com/wking/oci-runtime-config-validator/blob/cd0facf0ae1a6e6be7d8a20cac74b1299716b6b7/test/test_process.py#L134-L157